vault/logical_system_paths.go

```diff
@@ -1969,9 +2428,111 @@ func (b *SystemBackend) mountPaths() []*framework.Path {
+									Required:    true,
+								},
+								"default_lease_ttl": {
+									Type:        framework.TypeString,
+									Description: strings.TrimSpace(sysHelp["tune_default_lease_ttl"][0]),
+									Required:    true,
+								},
+								"force_no_cache": {
+									Type:     framework.TypeBool,
+									Required: true,
+								},
+								"token_type": {
+									Type:        framework.TypeString,
+									Description: strings.TrimSpace(sysHelp["token_type"][0]),
+									Required:    false,
+								},
+								"allowed_managed_keys": {
+									Type:        framework.TypeCommaStringSlice,
+									Description: strings.TrimSpace(sysHelp["tune_allowed_managed_keys"][0]),
+									Required:    false,
+								},
+								"allowed_response_headers": {
+									Type:        framework.TypeCommaStringSlice,
+									Description: strings.TrimSpace(sysHelp["allowed_response_headers"][0]),
+									Required:    false,
+								},
+								"allowed_request_headers": {
+									Type:        framework.TypeCommaStringSlice,
+									Description: strings.TrimSpace(sysHelp["allowed_response_headers"][0]),
+									Required:    false,
+								},
+								"options": {
+									Type:        framework.TypeKVPairs,
+									Description: strings.TrimSpace(sysHelp["tune_mount_options"][0]),
+									Required:    false,
+								},
+								"plugin_version": {
+									Type:        framework.TypeString,
+									Description: strings.TrimSpace(sysHelp["plugin-catalog_version"][0]),
+									Required:    false,
+								},
+								"external_entropy_access": {
+									Type:     framework.TypeBool,
+									Required: false,
+								},
+								"audit_non_hmac_request_keys": {
+									Type:     framework.TypeCommaStringSlice,
+									Required: false,
+								},
+								"audit_non_hmac_response_keys": {
+									Type:     framework.TypeCommaStringSlice,
+									Required: false,
+								},
+								"listing_visibility": {
+									Type:     framework.TypeString,
+									Required: false,
+								},
+								"passthrough_request_headers": {
+									Type:     framework.TypeCommaStringSlice,
+									Required: false,
+								},
+								"user_lockout_counter_reset_duration": {
+									Type:     framework.TypeInt64,
+									Required: false,
+								},
+								"user_lockout_threshold": {
+									Type:     framework.TypeInt64, // TODO this is actuall a Uint64 do we need a new type?
+									Required: false,
+								},
+								"user_lockout_duration": {
+									Type:     framework.TypeInt64,
+									Required: false,
+								},
+								"user_lockout_disable": {
+									Type:     framework.TypeBool,
+									Required: false,
+								},
+							},
+						}},
+					},
+				},
+				logical.UpdateOperation: &framework.PathOperation{
+					Callback: b.handleMountTuneWrite,
+					Responses: map[int][]framework.Response{
+						http.StatusNoContent: {{
```

Comment:
might have to test this, seems like the function returns a response struct, forget if that means 200 or 204

Reply:
probably just needs a `200` i updated it.